### ISSUE TITLE
news - author avatar name and title 

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#102]
When using Voice Over Utlity and tabbing to the author image on the news page, it reads as "avatar for". I think it should say something like "author avatar" or "author photo" since the user has to tab again for the name of the author to be read. I feel this way, the user isn't expecting the name to be said right away. 

Also, WAVE is alerting to all the author images as "image with title". This is happening on some of the other months that I checked in the dropdown. More info below:
![Screen Shot 2020-10-06 at 3 52 48 PM](https://user-images.githubusercontent.com/618307/95253195-0acfd100-07ec-11eb-8b76-c9c492b0b06e.png)

**Comment by kimisgold** [comment]
All the staff links should now have the same markup, including alt attribute that says "Profile pictures for [NAME]". That attribute should've read the same thing previously, so I imagine there was some kind of error that wasn't outputting the names correctly. It should be live now, and the following views are currently sharing markup:

* Who We Are
* Single project pages (under the contributor listing)
* News (index and archive)

They might need another look in case I forgot to adapt some styles specific to the context.

**Comment by IdaAkiwumi** [comment]
It looks like the individual project pages on all of them say "avatar for...". We should probably keep them consistent whether it's that or "picture for..."

**Comment by kimisgold** [comment]
Weird, they _should_ be consistent now. Can you hard refresh and double check? If you're still getting the same problem, can you link me to the page where you're seeing it?

**Comment by IdaAkiwumi** [comment]
It looks like it's just on these pages where I see "picture for..": https://rrchnm.org/who-we-are/ ;  https://rrchnm.org/news/ (archive page also)

and in contrast these pages and other author pages has "avatar for..." https://rrchnm.org/author/mills-kelly/ ; https://rrchnm.org/author/laquanda-walters-cooper/

**Comment by kimisgold** [comment]
Ahhh okay, I had missed the author page profile pictures. That should be updated now.
